com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/model/SupportedParameters.java

```diff
@@ -34,6 +34,7 @@
     public static final String GENERATED_PASSWORD = "generated-password";
     public static final String DEFAULT_IDLE_DOMAIN = "default-idle-domain";
     public static final String TIMESTAMP = "timestamp";
+    public static final String ENABLE_PARALLEL_DEPLOYMENTS = "enable-parallel-deployments";
```

Comment:
Why "deployments"? I thought we agreed upon "operations".

Reply:
For us, as developers it makes more sense to indeed change it to "operations", because we know the bg-deploy internally uses the app-deploy process to restart apps but for users I think it makes more sense to define it as "deployments" because they start a bg-deploy process and simply throughout the output they cannot make a difference between the prevalidation deployment and postvalidation "deployment". If I was a user, I would interpret the whole process as simply "deployment". And this parameter heavily impacts user experience. Wouldn't changing it to operations bring questions like "This is set to parallel ops, then why isn't the creation of services parallel too? It is an operation...". Same question applies for a lot of other stuff. What do you think?

Reply:
I can see where you're coming from. Some other people have also given alternative names, so let's all vote and decide the final name after the Scrum meeting tomorrow.